app/models/post.rb

```diff
@@ -1,3 +1,7 @@
 class Post < ApplicationRecord
-  belongs_to :author_id
+  belongs_to :user
```

Comment:
- [ ]  Great job so far but I think it would be great if you add here reference to the class with the foregin key so I recommend to add it here for the three relation since in this post table there is author_id reference to it 🤩✅ so please add it


### You can do somthing like this ⬇
`belongs_to :author, class_name: 'User'`